sample_app/main_linux.c

```diff
@@ -30,7 +30,8 @@
 #include <string.h>
 #include <unistd.h>
 
-#define EXIT_CODE_ERROR                1
+#define APP_EXIT_CODE_ERROR            1
```

Comment:
There are actually standardized constants for these, EXIT_SUCCESS and EXIT_FAILURE.

Reply:
Nice, done.